packages/core/components/MultiSelect/MultiSelect.tsx

```diff
@@ -67,77 +77,86 @@ const MultiSelect: React.FC<MultiSelectProps> = ({ section = null, subsection =
 
   const multiID = 'multiSelect_' + label
   return (
-    <div ref={multiSelectRef} className='cove-multiselect'>
+    <>
       {label && (
-        <span id={multiID} className='edit-label column-heading'>
+        <label className='edit-label column-heading' id={multiID + label} htmlFor={multiID}>
           {label}
-        </span>
+        </label>
       )}
+      <div ref={multiSelectRef} className='cove-multiselect'>
+        {tooltip && tooltip}
 
-      {tooltip && tooltip}
-
-      <div className='wrapper'>
-        <div className='selected'>
-          {selectedItems.map(item => (
-            <div key={item.value} aria-labelledby={label ? multiID : undefined}>
-              {item.label}
-              <button
-                aria-label='Remove'
-                onClick={e => {
-                  e.preventDefault()
-                  handleItemRemove(item)
-                }}
-                onKeyUp={e => {
-                  handleItemRemove(item, e)
-                }}
-              >
-                x
-              </button>
-            </div>
-          ))}
-          <button
-            aria-label={expanded ? 'Collapse' : 'Expand'}
-            aria-labelledby={label ? multiID : undefined}
-            className='expand'
-            onClick={e => {
-              e.preventDefault()
-              setExpanded(!expanded)
+        <div className='wrapper'>
+          <div
+            id={multiID}
+            onClick={() => {
+              if (!selectedItems.length) {
+                setExpanded(true)
+              }
             }}
+            className='selected'
           >
-            <Icon display={expanded ? 'caretDown' : 'caretUp'} style={{ cursor: 'pointer' }} />

```

Comment:
I'm assuming they don't want the caretUp here anymore for ux reasons.

Reply:
yes that was as per the prototype.